src/main/java/de/nononitas/plotborder/PlotBorder.java

```diff
@@ -90,7 +90,7 @@ public void onEnable() {
             return;
         }
         String plotVersion = Bukkit.getPluginManager().getPlugin("PlotSquared").getDescription().getVersion();
-        if (!plotVersion.startsWith("6.")) {
+        if (!plotVersion.startsWith("7.")) {
```

Comment:
If there are no API changes that needs to be implemented why not support v6 and v7?

Reply:
You are right, I will test it

Reply:
I have added Support back for 6. it works with the Version 6 fine.